apps/src/templates/studioHomepages/teacherHomepageV2/SectionList.tsx

```diff
@@ -86,6 +94,13 @@ export const SectionList: React.FC<SectionListProps> = ({
     [sections]
   );
 
+  // Update sectionsLoading state when asyncLoadComplete changes
+  React.useEffect(() => {
```

Comment:
Why do we need to use the state and a useEffect here instead of just using the redux value from `asyncLoadComplete`?